doc/lua_api.md

```diff
@@ -6666,6 +6666,8 @@ Environment access
       of the *active* mapgen setting `"mapgen_limit"`.
     * `chunksize` is an optional number. If it is absent, its value is that
       of the *active* mapgen setting `"chunksize"`.
+* `core.get_mapgen_chunksize()`
+    * Returns the currently active chunksize of the mapgen, as a vector.
```

Comment:
Please add doc that it's in blocks. I know this might be obvious for some, but having it explicit is just so much easier for the reader.